cloudera/dbt-impala
# cross-db macros

## Describe your changes
* add dbt core dependency for new macros
* copy over cross db macros
* add tests for cross db macros
* remove redundant current_timestamp macro
* fix macros for impala specific sql
* modify tests specific to impala (TestCastBoolToText)
* fix syntax issues with test sql from base package 

## Internal Jira ticket number or external issue link
DBT-498

## Testing procedure/screenshots(if appropriate):
Run the functional test:
python -m pytest tests/functional/adapter/test_utils.py -k
All tests except TestListAgg should pass. For TestListAgg, tests requiring order by and limit clauses will fail as these are not supported in impala see: https://issues.apache.org/jira/browse/IMPALA-5545

## Checklist before requesting a review
